src/ape/managers/query.py

```diff
@@ -1,13 +1,60 @@
-from typing import Dict, Optional
+from functools import partial
+from typing import Any, Dict, Optional
 
 import pandas as pd
+from pydantic import BaseModel
 
 from ape.api import QueryAPI, QueryType
+from ape.api.query import BlockQuery, _BaseQuery
 from ape.exceptions import QueryEngineError
 from ape.plugins import clean_plugin_name
 from ape.utils import ManagerAccessMixin, cached_property
 
 
+def get_columns_from_item(query: _BaseQuery, item: BaseModel) -> Dict[str, Any]:
```

Comment:
```suggestion
def _get_columns_from_item(query: _BaseQuery, item: BaseModel) -> Dict[str, Any]:
```

this will ensure this method stays out of the docs

Reply:
This method is non-optimal anyways, @NotPeopling2day discovered a nicer way via Pandas